src/JavierLeon9966/VanillaElytra/VanillaElytra.php

```diff
@@ -81,6 +81,9 @@ public function onPlayerMove(PlayerMoveEvent $event): void{
 			return;
 		}
 
+		if($player->isOnGround()){
+			$player->toggleGlide(false);
```

Comment:
This should be `setGliding()`

Reply:
I used toggleGlide() to get the same behavior as before 1.20.10 (calling PlayerToggleGlideEvent)
still want to change it to setGliding()?

Reply:
Hmm, right